server/views/partials/profileBanner/macros/tabLinksMacro.njk

```diff
@@ -3,16 +3,24 @@
     {% for tabLink in tabLinks %}
       {% if tabLink.active %}
         <li class="govuk-heading-s hmpps-profile-tab-link-selected" data-qa="active-tab">
+          <span
```

Comment:
`href` isn't valid for a `<span>` and the link classes don't make any sense if it's not a link, so they can go too and just leave...
```suggestion
          <span data-qa="{{tabLink.label}}">{{ tabLink.label }}</span>
```